spec/features/invite_user_features_spec.rb

```diff
@@ -1,37 +1,44 @@
 require 'spec_helper'
 
 feature "Invite a user" do
-  scenario "User successfully sends and accepts an invitation" do
-    john = Fabricate(:user)
-    sign_in(john)
-    click_on "Welcome, #{john.full_name}"
-    click_on "Invite a friend"
-    fill_in "Friend's Name", with: 'Joe Smith'
-    fill_in "Friend's Email Address", with: 'jsmith@gmail.com'
-    click_on "Send invitation"
+  scenario "User successfully sends and accepts an invitation", { js: true } do
+    VCR.use_cassette('create_stripe_charge') do
```

Comment:
I suspect it is possible that your failing spec could come from the fact that you are sharing this cassette. I would let VCR handle the cassette naming itself.